### DISCUSSION TITLE
What the heck is this checkbox?! lol

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [ScratchAddons/ScratchAddons#257] [Help] [answered]
What is the checkbox on the top at the far right? lol
![image](https://user-images.githubusercontent.com/61319150/92761594-780c5580-f357-11ea-9d1d-bbc8d77ac046.png)

**Comment by BoomerScratch** [comment]
Probably the text doesn't show up :lol:

**Comment by Explosion-Scratch** [comment]
> Probably the text doesn't show up :lol:

What does the text say then? :P

**Comment by jeffalo** [comment, marked as answer]
replace stage?

**Comment by Explosion-Scratch** [comment]
> replace stage?

Oh, lol, the checkboxes are on the _right_ of the text. I for some reason thought it was the other way around.

**Comment by jeffalo** [comment]
perhaps this is a UX issue

**Comment by WorldLanguages** [comment]
Lol we should probably center the text

**Comment by WorldLanguages** [comment]
(or align it to the right)

**Comment by towerofnix** [comment]
Perhaps a light background transition could be applied when hovering over individual option tiles?

**Comment by just-a-hriday** [comment]
Hey that addon doesn't exist. Is it removed?

**Reply by Samq64** [reply]
I have never seen that addon either. Considering the post date, it was probably removed and then later replaced with the TurboWarp button addon.

**Reply by TheColaber** [reply]
It was an addon that was orginally added but removed due to having so many bugs.

**Reply by just-a-hriday** [reply]
The find recommendation from instructions or notes and credits thing seems interesting. Can it be added to `turbowarp-player`?

**Reply by Samq64** [reply]
TurboWarp already implements this with a stage comment.

**Reply by just-a-hriday** [reply]
> TurboWarp already implements this with a stage comment.
Oh yeah, I totally forgot about that.
